### PR TITLE
removed "< and  />"

### PR DESCRIPTION
**Pull request by utle**
Remove < and /> from the metatype properties.

### CODE REVIEW

**Comment by utle** [comment]
#build

File: dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -5,43 +5,43 @@
 #NLS_MESSAGEFORMAT_NONE
 
 authFilter=Authentication Filter
-authFilter.desc= Specifies a selection rule that represents conditions that are matched against the HTTP request headers. If all conditions are met, the HTTP request is selected for the authentication.
+authFilter.desc= Specifies a selection rule that represents conditions that are matched against the HTTP request headers. If all conditions are met, the HTTP request is selected for the authentication. Match Type contains and notContain support a piped list of values.
```
Comment (dmuelle) [inline dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties]:
```suggestion
authFilter.desc= Specifies a selection rule that represents conditions that are matched against the HTTP request headers. If all conditions are met, the HTTP request is selected for the authentication. Match Type contains and notContain support a piped list of values.
```
```suggestion
authFilter.desc= Specifies a selection rule that represents conditions that are matched against the HTTP request headers. If all conditions are met, the HTTP request is selected for the authentication. When the value for the matchType attribute is notContain or contains, the filter supports a piped list of values.
```

**Review by dmuelle — COMMENTED** [review]
ID review- made a suggestion

**Comment by utle** [comment]
#build

**Review by dmuelle — APPROVED** [review]

**Comment by wraschke** [comment]
Test comment; please ignore.

**Comment by wraschke** [comment]
Another test

**Comment by wraschke** [comment]
Hello, world.

**Comment by wraschke** [comment]
#libby

**Comment by wraschke** [comment]
Test

**Comment by wraschke** [comment]
Yellow daffodils swaying

**Comment by utle** [comment]
The personal build have two failures that are not cause this PR code change.

**Pull request merged**